### ISSUE TITLE
license info?

### ISSUE THREAD

**Issue by ITwrx** [fwartner/maps#2]
I'm not seeing a license for this project nor BIOSTALL's project:
https://github.com/BIOSTALL/CodeIgniter-Google-Maps-V3-API-Library/issues/34

Not having a license introduces risk for people who want to use this project in their projects. If this is Free Software, technically speaking, it needs a license to designate it as such. Even if the author(s) don't care if some company closes their fork up, if this project doesn't even have a MIT license then that hypothetical company could say this project(and subsequently all ours) infringe their proprietary fork.

I'm not saying that's super likely to happen but downstream devs wouldn't be doing their due diligence to use it without knowing it's license.

Thanks in advance.

**Comment by fwartner** [comment]
It was just a fork for one of my projects.. I just changed a few things to make it work in L5.1 and wasn´t deleting any files..

**Comment by ITwrx** [comment]
I thought that might be why you didn't license yours. I had the same decision to make on something i put on github. I had changed even less than you have so i didn't want people to think i was trying to relicense the original author's work, claim credit, etc. I licensed my fork anyways for the reasons i mentioned above. If people wanted the original project under it's license they could easily use that. my version, with my minor changes still needed a license and would be licensed AGPL. I would encourage you to license this fork using the license of your choice. 

Thanks.
